config/enjin-platform.php

```diff
@@ -230,6 +230,8 @@
         'adapter' => PlatformQrAdapter::class,
         'size' => env('QR_CODE_SIZE', 512),
         'format' => env('QR_CODE_FORMAT', 'png'),
+        'image' => env('QR_CODE_IMAGE_URL', __DIR__ . '/../resources/images/enjin-logo.jpg'),
```

Comment:
Please could we make this configurable so self-hosted users can override and choose their own logos, or not have an image at all.